examples/todo-list/src/AddTodoTask.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { useStore } from "./store";
+import { FcTodoList } from "react-icons/fc";
```

Comment:
Is necessary this import?

Reply:
deleted, was from a deleted icon that we're not using anymore